src/python_freeipa/client.py

```diff
@@ -566,6 +576,17 @@ def user_del(self, username, skip_errors=False, soft_delete=False):
         }
         return self._request('user_del', username, params)
 
+    def user_undel(self, username):
+        """
+        Undelete a user.
+
+        :param username: User login.
+        :type username: string
+        :param skip_errors: Continuous mode: Don't stop on errors.
```

Comment:
Hm, this param is not defined in method signature. Could you please either remove this param from docstring or add it to the method signature?